<?xml version="1.0"?>
<launch>
  <node name="object_eval_pose" pkg="ros_votepose" type="votepose_eval_node" cwd="node">
  
  <param name="listname_path" value="/home/hoang/Orebro-Study/Manuscripts/ICCV2021/data/experiment/brick/list_names_full.txt"/>
  <param name="model_path" value="/home/hoang/Orebro-Study/Manuscripts/ICCV2021/data/experiment/brick/spare.ply"/>
  
  <param name="pc_dir" value="/home/hoang/Orebro-Study/Manuscripts/ICCV2021/data/experiment/brick/pointcloud/"/>
  <param name="gt_dir" value="/home/hoang/Orebro-Study/Manuscripts/ICCV2021/data/experiment/brick/gt/"/>
  <param name="baseline_dir" value="/home/hoang/Orebro-Study/Manuscripts/ICCV2021/data/experiment/brick/predicted/baseline/"/>
  <param name="proposed_dir" value="/home/hoang/Orebro-Study/Manuscripts/ICCV2021/data/experiment/brick/predicted/proposed/"/>
  
  <param name="z_max" value="0.5"/>
  <param name="z_min" value="-0.3"/>
  <param name="ratio_error_max" value="0.2"/>
  
  </node>
   <!-- Show in Rviz   -->
  <node name="rviz1" pkg="rviz" type="rviz" args="-d $(find ros_votepose)/launch/bop_eval_scene.rviz"/>
  <node name="rviz2" pkg="rviz" type="rviz" args="-d $(find ros_votepose)/launch/bop_eval_gt.rviz"/>
  <node name="rviz3" pkg="rviz" type="rviz" args="-d $(find ros_votepose)/launch/bop_eval_baseline.rviz"/>
  <node name="rviz4" pkg="rviz" type="rviz" args="-d $(find ros_votepose)/launch/bop_eval_proposed.rviz"/>

</launch>